### ISSUE TITLE
Insert f5 - form list window showing incorrect labels of the tab elements

### ISSUE THREAD

**Issue by mounikaseelam** [FreedomScientific/VFO-standards-support#461]
### Summary

> Insert f5 - form list window showing incorrect labels of the tab elements
> when we have tabs widget with proper accessibility related attribute, and when we press insert f5
the list showing tabs label 
for active tab the label is shown correct , but for the rest of the tabs , the label is shown with some other text 
example : if tab name is "**Agnes Obe**" , but in the list it is shown as **"test1 Agnes Obe"**
here test1 is the heading in the first tabpanel 
> Example:
> 1. Go to https://www.w3schools.com/code/tryit.asp?filename=GJQEW3U0I8OY
> 2. press inser f5

### Expected result

> labels of the tab should be shown correctly

### Actual result

> tab labels are shown with concat of other text from the active tab tabpanel

### Example

> https://www.w3schools.com/code/tryit.asp?filename=GJQEW3U0I8OY

### Additional Information

using Jaws 2020
OS window 10
browser Chrome - Version 84.0.4147.125

<img width="792" alt="2020-10-15_20-17-26" src="https://user-images.githubusercontent.com/37592390/96151590-34de6d00-0f29-11eb-9c21-43016bacdbec.png">

**Comment by mounikaseelam** [comment]
Please any one help me on this issue

**Comment by JAWS-test** [comment]
If the ARIA role tab is used, the tabpanel should not be in the tablist (see https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html)

Probably the following is more suitable for you: https://www.w3.org/TR/wai-aria-practices-1.1/examples/disclosure/disclosure-faq.html

**Comment by mounikaseelam** [comment]
Thankyou @JAWS-test 
The mark up is like what i have shared (https://www.w3schools.com/code/tryit.asp?filename=GJQEW3U0I8OY), but the ui looks like the tabs widget like below image, so we used design pattern of tab widget . with the existing markup is there any solution to get rid of that issue as there is no scope of modifying the existing markup 
<img width="308" alt="2020-10-16_19-21-45" src="https://user-images.githubusercontent.com/37592390/96266987-eee6df00-0fe4-11eb-8d99-1740c5c9607c.png">

**Comment by JAWS-test** [comment]
If you use tabs, the source code must also be correct. tabpanel must come after tablist and not within it. This is not a JAWS bug, but a bug in the source code. Another problem with your layout is the reading order, which must be correct according to [WCAG 1.3.2](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html), but this is not the case here (all three tabs must come before tabpanel in source code)
